---
permalink: /vitro/
title: Labwork
---

For a thorough study of geobiology (or environmental microbiology) I believe it is vital to combine observations from the field with experiments in the lab.
More recently, a third component can be added to the list: sequence analysis, and bioinformatics more broadly. On the "in Situ", "in Vitro", 
and "in Silico" pages of this site, I highlight some of my research and teaching efforts in these three components of geobiology.     
   
This page highlights my efforts in "wet" labwork.      
    
    
### Teaching   
   
**GeoBio course molecular biology module**   
The International Geobiology course brings together students from fields adjacent to geobiology (Bio/Chem/Geo) and gives them a taste of "the other side".
As such, the Caltech hosted installments feature a whirlwind of different lab techniques, from nanoSIMS, to petrography, to molecular biology. For the last
three years, I have been responsible for teaching the molecular biology section, which involves DNA extraction, a 16S rRNA gene PCR, and gel 
electrophoresis. The follow-up barcoding PCR and pooling is then done by Dr. Stephanie Connon, and the turnaround time of the amplicon sequencing 
is short enough that the students can then analyse their results in the final weeks of the course. The module starts with a refresher lecture on how DNA
extraction, PCR, and gel electrophoresis work, and why we use the rRNA genes for amplicon sequencing. Using the Powersoil kit, and the highly processive 
Q5 polymerase, the labwork fits in a single day, with the students extracting 6 samples each.   
   
    
**Undergraduate (micro)biology courses** 
    
    
### Research   
   
**Experimental design for metagenomics studies**   
in progress
    
**Co-cultivation of "unculturable" microorganisms**  
The rapid advances in DNA sequencing and analysis of sequencing data have been transformative for our understanding of microbial diversity. The 
developement of genome resolved metagenomics has added many new branches to the tree of life, but one of the major challenges in the interpretation 
of this wealth of data is the lack of characterized organisms in these new branches. The confidence of our metabolic predictions correlates with the 
evolutionary distance to microorganisms that have been characterized in the lab. Obtaining cultured representatives across the tree of life will be
key for continued growth of our knowledge. To achieve this, we must ask ourselves: why are organisms thought to be "unculturable"?     
One answer to this question could be growth rate, which is hard to tackle. But it is not the only possible answer. Another possibility is lack of 
knowledge on growth requirements of uncultured organisms. One way we can attempt to migitate this, is letting other members of the microbial community
do that work for us. Using knowledge on the structural organization of microbial communities, we can enrich microbes with their natural partners 
in reduced communities. Those reduced communities will be a valuable resource for eludication of the principles behind microbial "unculturability".   
    
**Outlook: using sequence information to prioritize protein characterization**    
in progress     
